dashboard/test/ui/features/share_buttons.feature

```diff
@@ -0,0 +1,23 @@
+@as_student
+  # need @as_student to log in in order to get to Game Lab since it's a droplet game
+
+Feature: Share Buttons
+  # Making How It Works button show up in Sprite Lab and not in Game Lab
+
+Scenario: How It Works Button appears for Sprite Lab share page
```

Comment:
This scenario should be indented one more tab to align with the other scenario.

Reply:
Woops! Will go ahead and correct that. Thank you for letting me know!